### DISCUSSION TITLE
job service ignores entrypoint

### DISCUSSION THREAD

**Discussion by trajano** [PremoWeb/chadburn#83] [General]
The `command` parameter should be applied as `arguments` in a service but it appears to be replacing `entrypoint`.

Also it would be nice to have a configuration `entrypoint` to explicitly override entrypoint

**Comment by maietta** [comment]
Hi @trajano.

I am planning a total rewrite and will put this close to the top of my list.

When I forked the Ofelia project, I didn't realize just how many issues people were dealing with. I will be carving out about a month's time sometime this Spring to do the rewrite.

Thank you for the suggestions. I will leave this issue open.
